gateway/src/main/java/io/zeebe/gateway/GrpcServerMetricsCleaner.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe.gateway;
+
+import io.prometheus.client.Collector;
+import io.prometheus.client.CollectorRegistry;
+import java.lang.reflect.Field;
+import java.util.Optional;
+import me.dinowernli.grpc.prometheus.MonitoringServerInterceptor;
+
+/**
+ * Hotfix for deregistring metrics from gRPC server metrics to run tests without duplicate metrics
+ */
+public class GrpcServerMetricsCleaner {
+
+  private static final CollectorRegistry COLLECTOR_REGISTRY = CollectorRegistry.defaultRegistry;
```

Comment:
How about disabling metrics in tests instead of this beautiful hack :man_shrugging: :sweat_smile:

Reply:
I mean a configuration which can enable and disable metrics makes somehow sense right?